.github/workflows/.github/workflows/linters.yml

```diff
@@ -0,0 +1,48 @@
+name: Linters
+
+on: pull_request
+
```

Comment:
- [ ] You set up the linter in the wrong directory that's why there are not running. Please use the correct directory for linter.
   - Please change the directory from .`github/workflows/.github/workflows/linters.yml` to `.github/workflows/linters.yml`

Reply:
Hi there, thank you for your help. 

So I changed the directory as mentioned, but am now having issues with stylelint and webhint. 

I will also changed the rest as mentioned and submit again

Reply:
Hi @thecodechaser I did it, but no fix? Pls help!

Reply:
@CrystallineButterfly Hello, That's linter all about, we need to fix these errors so we follow the best practices for our code.

Please contact me on slack if you need further explanation. You can find me on slack @thecodechaser

Reply:
I fully agree and understand; However @thecodechaser How do I solve the Webhint and Stylint failures to 100% This exercise! Or is it ready to send for another review as of now?

Reply:
@CrystallineButterfly  No it is not. All checks should be passed before you go for another code review

Reply:
@thecodechaser How do i solve those checks?